static/_app/tools/idle-idle-gamedev/css/style.css

```
@@ -0,0 +1,206 @@
+    opacity: 0.25;
+}
+.card{
+    background-color: #555;
+    height: 100%;
+    justify-content: space-between;
+}
+.skill-row{
+    margin-bottom: 1vh;
+}
+.modal-container{
+    background: linear-gradient(0,#303030,#424242);
```

Biome (2.1.2)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)